### DISCUSSION TITLE
Feature: Add types in batch get

### DISCUSSION THREAD

**Discussion by ShivamJoker** [sensedeep/dynamodb-onetable#337] [Q&A] [answered]
It would be great to get typescript types in the batchGet response as well.

Right now it returns an array of unknown which is not helpful at all.

![image](https://user-images.githubusercontent.com/23727670/169645979-2e28eb4a-6ffc-44e4-9b96-5473515b2af1.png)

**Comment by mobsense** [comment, marked as answer]
I've added an issue / enhancement for this
